presentation/src/main/java/com/mashup/gabbangzip/sharedalbum/presentation/ui/vote/PhotoCard.kt

```diff
@@ -35,17 +49,58 @@ fun PhotoCard(
                         state = swipeState,
                         onDragLiked = {
                             swiped.value = true
-                            onSwiped(PhotoVoteResult.LIKE, photo)
+                            onSwiped(PhotoVoteType.LIKE, photo)
                         },
                         onDragDisliked = {
                             swiped.value = true
-                            onSwiped(PhotoVoteResult.DISLIKE, photo)
+                            onSwiped(PhotoVoteType.DISLIKE, photo)
                         },
                     ),
                 contentAlignment = Alignment.Center,
             ) {
                 content()
             }
         }
+
+        LaunchedEffect(voteClickInfo.index == currentIndex) {
+            when (voteClickInfo.type) {
+                PhotoVoteType.LIKE -> {
+                    isSwiped.value = true
+                    onVoteClickSwiped(PhotoVoteType.LIKE, photo)
+                    swipeState.liked(coroutineScope)
+                    delay(VoteConstant.CARD_DELAY_TIME)
+                    swiped.value = true
+                    isSwiped.value = false
+                }
+
+                PhotoVoteType.DISLIKE -> {
+                    isSwiped.value = true
+                    onVoteClickSwiped(PhotoVoteType.DISLIKE, photo)
+                    swipeState.disliked(coroutineScope)
+                    delay(VoteConstant.CARD_DELAY_TIME)
+                    swiped.value = true
+                    isSwiped.value = false
+                }
+
+                else -> Unit
+            }
+        }
+
+        // 버튼 클릭후 1초 딜레이 되는동안 카드 스와이프 막기 위함
+        if (isSwiped.value) {
```

Comment:
```suggestion
        if (swipeEnabled.value) {
```
`isSwiped` 가 스와이프가 가능하지를 판단하기 위해 있는 함수이다보니 swipeEnabled 같은 이름이 조금 더 어울릴 것 같다는 생각이 드는데 어떤 것 같아?!

Reply:
너무 좋은데? 훨씬 명확한거같아..! 바로 수정하러 갑니다 ㅋㄷㅋㄷ